app/components/CrashIsTemporaryBanner.tsx

```diff
@@ -0,0 +1,66 @@
+import { useRouter } from "next/navigation";
+import Alert from "react-bootstrap/Alert";
+import Button from "react-bootstrap/Button";
+import Spinner from "react-bootstrap/Spinner";
+import { FaTrash, FaTriangleExclamation } from "react-icons/fa6";
+import AlignedLabel from "./AlignedLabel";
+import { useMutation } from "@/utils/graphql";
+import { DELETE_CRIS_CRASH } from "@/queries/crash";
+
+interface CrashIsTemporaryBannerProps {
+  crashId: number;
+}
+
+/**
+ * Banner that alerts the user when viewing a "temporary" crash record, aka
+ * a crash record that was manually created through the UI
+ */
+export default function CrashIsTemporaryBanner({
+  crashId,
+}: CrashIsTemporaryBannerProps) {
+  const { mutate, loading: isMutating } = useMutation(DELETE_CRIS_CRASH);
```

Comment:
@johnclary Does this mutation need to soft delete the units too? I can't remember exactly how the totals are aggregated, but I am seeing `is_deleted = false` on the temp units after deleting the temp crash.

Reply:
i was wondering the same thing too, and for any persons created as well